MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -168,16 +171,17 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
         /// Submit a sprite for drawing in the current batch.
         /// </summary>
         /// <param name="texture">A texture.</param>
-        /// <param name="position">An optional drawing location on screen.</param>
-        /// <param name="destinationRectangle">An optional drawing bound.</param>
+        /// <param name="position">The drawing location on screen or null if <param name="destinationRectangle"> is used.</param></param>
```

Comment:
I think this should use a `<paramref>` to reference the destinationRectangle parameter.